common-types/index.d.ts

```diff
@@ -131,3 +131,17 @@ interface CandidateDeciderInfo {
   readonly uuid: string;
   isOpen: boolean;
 }
+
+interface DevPortfolio {
+  name: string;
+  deadline: number;
+  earliestValidDate: number;
+  submissions: DevPortfolioSubmission[];
+}
+
+interface DevPortfolioSubmission {
+  member: IdolMember;
+  openedPR: string;
+  reviewedPR: string;
```

Comment:
Would it be more convenient to be able to submit multiple PRs for the opened or reviewed requirement at once, in case one of them is determined not to be valid?

Reply:
Hmmm good point.... it might be good to keep track of submission history too. I think the interface can stay the same but we keep all submissions and we just treat the most recently submitted opened PR and the reviewed PR as the most complete submission?

Reply:
lol jk that isn't gonna work